OpenRA.Mods.Common/Traits/BlocksProjectiles.cs

```diff
@@ -13,10 +13,11 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
-	// TODO: Add functionality like a customizable Height that is compared to projectile altitude
 	[Desc("This actor blocks bullets and missiles with 'Blockable' property.")]
 	public class BlocksProjectilesInfo : UpgradableTraitInfo
 	{
+		public readonly WDist Height = WDist.FromCells(6);
```

Comment:
> Note that one cell will be less high than some arcing projectiles (artillery, etc).

Arcing projectiles in the shipping mods have `Blockable: false` anyway, so that shouldn't matter.